src/fio/fioEngine.js

```diff
@@ -15,13 +15,13 @@ import {
 } from 'edge-core-js/types'
 
 import { CurrencyEngine } from '../common/engine.js'
-import { asyncWaterfall, getDenomInfo } from '../common/utils'
+import { asyncWaterfall, getDenomInfo, shuffleArray } from '../common/utils'
```

Comment:
@paullinator I removed my shuffle method, but have a question, in my method I shuffled copy of the array, not actual array
```
export const shuffle = (array: any[]): any[] => {
  const shuffledArr = [...array]
```
should we add this to common shuffle method?